tests/kernel/install_ltp.pm

```diff
@@ -215,9 +215,34 @@ sub install_from_git {
 }
 
 sub install_from_repo {
-    my ($tag) = @_;
-    zypper_call('in qa_test_ltp', dumb_term => 1);
-    script_run 'rpm -q qa_test_ltp | tee /opt/ltp_version';
+    my ($tag)  = @_;
+    my $is_sle = is_sle;
```

Comment:
Is `$is_sle` variable really needed? I think `is_sle` can be used at all places.

Reply:
Now I see, it was added after comment from @SeroSun to cache result. This approach not systematic, it just adding clutter decreasing readability of code and I doubt that we have measurable performance hit.

Reply:
Uh, previous version @SeroSun objected about using `is_sle` and suggested use `$is_sle` :). Syntactic details are also valuable and needed, but IMHO review is more about checking if something important was overlooked.